sanityv3/schemas/objects/fullWidthVideo.tsx

```diff
@@ -43,14 +43,11 @@ export default {
         list: [
           { title: 'fullscreen', value: 'fullscreen' },
           { title: 'narrow', value: 'narrow' },
-          { title: '16:9', value: '16:9' },
-          { title: '9:16', value: '9:16' },
           { title: '2:1', value: '2:1' },
-          { title: '1:1', value: '1:1' },
         ],
         layout: 'dropdown',
       },
-      initialValue: '16:9',
+      initialValue: 'fullscreen',
```

Comment:
ratios are not working as expected. for example, narrow is being displayed as full screen:
![image](https://github.com/equinor/energyvision/assets/23337266/2e4a2028-d428-4946-a5ed-6ea6bc8a5bf5)

I recommend looking at the full width hero component, to see how logic was implemented there. it would also be nice if the values/titles are consistent with the header when it comes to letter case (fullScreen vs fullscreen) etcs:

![image](https://github.com/equinor/energyvision/assets/23337266/a2be3a4a-0eca-47fb-a05f-28c797dcbe99)

Reply:
Changes should also be added to the /studio folder, since we have not yet migrated to v3

Reply:
✅ Renamed values and titles to be more consistent 
✅ Narrow banner is now actual narrow and not only in my cached version x)
✅ Rewrote fullwidthvideo styling logic

about the full width hero component, i felt it would be overkill to create different functions depending on the ratio as done here. Let me know your thoughts of how it is now @fernandolucchesi

Reply:
> Changes should also be added to the /studio folder, since we have not yet migrated to v3

will do this asap!

Reply:
✅ Added to v2 studio